[2021-12-22 17:44:46,225] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline1.super_resolution scheduled__2021-12-20T00:33:00+00:00 [queued]>
[2021-12-22 17:44:46,228] {taskinstance.py:1035} INFO - Dependencies all met for <TaskInstance: data_pipeline1.super_resolution scheduled__2021-12-20T00:33:00+00:00 [queued]>
[2021-12-22 17:44:46,228] {taskinstance.py:1241} INFO - 
--------------------------------------------------------------------------------
[2021-12-22 17:44:46,228] {taskinstance.py:1242} INFO - Starting attempt 1 of 4
[2021-12-22 17:44:46,228] {taskinstance.py:1243} INFO - 
--------------------------------------------------------------------------------
[2021-12-22 17:44:46,237] {taskinstance.py:1262} INFO - Executing <Task(BashOperator): super_resolution> on 2021-12-20 00:33:00+00:00
[2021-12-22 17:44:46,238] {standard_task_runner.py:52} INFO - Started process 33079 to run task
[2021-12-22 17:44:46,240] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'data_pipeline1', 'super_resolution', 'scheduled__2021-12-20T00:33:00+00:00', '--job-id', '192', '--raw', '--subdir', '/home/myamya/project/dockers/airflow/dags/hello_world.py', '--cfg-path', '/tmp/tmpnserumwr', '--error-file', '/tmp/tmp20gzn0ik']
[2021-12-22 17:44:46,240] {standard_task_runner.py:77} INFO - Job 192: Subtask super_resolution
[2021-12-22 17:44:46,258] {logging_mixin.py:109} INFO - Running <TaskInstance: data_pipeline1.super_resolution scheduled__2021-12-20T00:33:00+00:00 [running]> on host myamya
[2021-12-22 17:44:46,302] {taskinstance.py:1412} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=yangjae
AIRFLOW_CTX_DAG_ID=data_pipeline1
AIRFLOW_CTX_TASK_ID=super_resolution
AIRFLOW_CTX_EXECUTION_DATE=2021-12-20T00:33:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-12-20T00:33:00+00:00
[2021-12-22 17:44:46,303] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2021-12-22 17:44:46,303] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'docker run -it -v /home/myamya/project/image_files/:/home/myamya/project/image_files/ sure:0.3 /bin/bash /home/myamya/project/init.sh ']
[2021-12-22 17:44:46,311] {subprocess.py:85} INFO - Output:
[2021-12-22 17:44:47,908] {subprocess.py:89} INFO - /opt/conda/lib/python3.8/site-packages/torch/cuda/__init__.py:52: UserWarning: CUDA initialization: Found no NVIDIA driver on your system. Please check that you have an NVIDIA GPU and installed a driver from http://www.nvidia.com/Download/index.aspx (Triggered internally at  /opt/conda/conda-bld/pytorch_1603729096996/work/c10/cuda/CUDAFunctions.cpp:100.)
[2021-12-22 17:44:47,908] {subprocess.py:89} INFO -   return torch._C._cuda_getDeviceCount() > 0
[2021-12-22 17:44:48,913] {subprocess.py:89} INFO - Downloading: "https://github.com/xinntao/facexlib/releases/download/v0.1.0/detection_Resnet50_Final.pth" to /opt/conda/lib/python3.8/site-packages/facexlib/weights/detection_Resnet50_Final.pth
[2021-12-22 17:44:48,913] {subprocess.py:89} INFO - 
[2021-12-22 17:44:51,757] {subprocess.py:89} INFO -   0% 0.00/104M [00:00<?, ?B/s]  0% 216k/104M [00:00<00:50, 2.16MB/s]  1% 1.18M/104M [00:00<00:38, 2.82MB/s]  4% 4.41M/104M [00:00<00:27, 3.86MB/s]  8% 7.88M/104M [00:00<00:19, 5.28MB/s] 13% 13.2M/104M [00:00<00:13, 7.24MB/s] 18% 18.6M/104M [00:00<00:09, 9.81MB/s] 21% 22.2M/104M [00:00<00:06, 12.5MB/s] 26% 27.6M/104M [00:00<00:04, 16.3MB/s] 32% 33.0M/104M [00:00<00:03, 20.8MB/s] 36% 37.5M/104M [00:01<00:02, 24.3MB/s] 41% 42.6M/104M [00:01<00:02, 29.0MB/s] 46% 48.0M/104M [00:01<00:01, 34.0MB/s] 51% 52.8M/104M [00:01<00:01, 35.2MB/s] 56% 58.2M/104M [00:01<00:01, 39.8MB/s] 61% 63.6M/104M [00:01<00:00, 43.7MB/s] 66% 68.6M/104M [00:01<00:00, 42.2MB/s] 71% 74.0M/104M [00:01<00:00, 45.7MB/s] 75% 78.8M/104M [00:01<00:00, 43.5MB/s] 81% 84.1M/104M [00:02<00:00, 46.6MB/s] 86% 89.5M/104M [00:02<00:00, 49.3MB/s] 90% 94.5M/104M [00:02<00:00, 47.1MB/s] 95% 99.3M/104M [00:02<00:00, 48.1MB/s]100% 104M/104M [00:02<00:00, 44.5MB/s]
[2021-12-22 17:44:51,907] {subprocess.py:89} INFO - Downloading: "https://github.com/TencentARC/GFPGAN/releases/download/v0.2.0/GFPGANCleanv1-NoCE-C2.pth" to /opt/conda/lib/python3.8/site-packages/gfpgan/weights/GFPGANCleanv1-NoCE-C2.pth
[2021-12-22 17:44:51,907] {subprocess.py:89} INFO - 
[2021-12-22 17:45:00,330] {subprocess.py:89} INFO -   0% 0.00/332M [00:00<?, ?B/s]  0% 232k/332M [00:00<02:48, 2.07MB/s]  1% 1.91M/332M [00:00<02:03, 2.81MB/s]  1% 4.62M/332M [00:00<01:29, 3.85MB/s]  2% 7.93M/332M [00:00<01:05, 5.23MB/s]  4% 13.2M/332M [00:00<00:46, 7.19MB/s]  5% 17.2M/332M [00:00<00:34, 9.55MB/s]  6% 21.2M/332M [00:00<00:26, 12.4MB/s]  8% 25.2M/332M [00:00<00:20, 15.7MB/s]  9% 29.4M/332M [00:00<00:16, 19.5MB/s] 10% 33.4M/332M [00:01<00:13, 23.2MB/s] 11% 37.6M/332M [00:01<00:11, 27.0MB/s] 13% 41.6M/332M [00:01<00:10, 30.3MB/s] 14% 45.8M/332M [00:01<00:09, 33.2MB/s] 15% 50.0M/332M [00:01<00:08, 35.9MB/s] 16% 54.2M/332M [00:01<00:07, 38.0MB/s] 18% 58.4M/332M [00:01<00:07, 39.8MB/s] 19% 62.7M/332M [00:01<00:06, 41.0MB/s] 20% 67.0M/332M [00:01<00:06, 42.1MB/s] 21% 71.3M/332M [00:01<00:06, 42.9MB/s] 23% 75.5M/332M [00:02<00:06, 43.4MB/s] 24% 79.9M/332M [00:02<00:06, 44.1MB/s] 25% 84.2M/332M [00:02<00:05, 44.3MB/s] 27% 88.7M/332M [00:02<00:05, 45.0MB/s] 28% 93.0M/332M [00:02<00:05, 45.0MB/s] 29% 97.5M/332M [00:02<00:05, 45.7MB/s] 31% 102M/332M [00:02<00:05, 45.5MB/s]  32% 106M/332M [00:02<00:05, 46.1MB/s] 33% 111M/332M [00:02<00:05, 46.0MB/s] 35% 115M/332M [00:02<00:04, 46.4MB/s] 36% 120M/332M [00:03<00:04, 46.4MB/s] 37% 124M/332M [00:03<00:04, 46.7MB/s] 39% 129M/332M [00:03<00:04, 46.6MB/s] 40% 133M/332M [00:03<00:04, 47.0MB/s] 41% 138M/332M [00:03<00:04, 47.2MB/s] 43% 142M/332M [00:03<00:04, 47.1MB/s] 44% 147M/332M [00:03<00:04, 47.4MB/s] 46% 152M/332M [00:03<00:04, 47.3MB/s] 47% 156M/332M [00:03<00:03, 47.8MB/s] 48% 161M/332M [00:03<00:03, 47.6MB/s] 50% 165M/332M [00:04<00:03, 47.9MB/s] 51% 170M/332M [00:04<00:03, 47.9MB/s] 53% 175M/332M [00:04<00:03, 48.1MB/s] 54% 179M/332M [00:04<00:03, 48.2MB/s] 55% 184M/332M [00:04<00:03, 48.4MB/s] 57% 189M/332M [00:04<00:07, 20.6MB/s] 58% 193M/332M [00:05<00:05, 25.0MB/s] 59% 198M/332M [00:05<00:04, 29.1MB/s] 61% 203M/332M [00:05<00:04, 33.4MB/s] 62% 207M/332M [00:05<00:03, 36.7MB/s] 64% 212M/332M [00:05<00:03, 39.9MB/s] 65% 217M/332M [00:05<00:02, 42.0MB/s] 67% 221M/332M [00:05<00:02, 44.2MB/s] 68% 226M/332M [00:05<00:02, 45.3MB/s] 69% 231M/332M [00:05<00:02, 46.6MB/s] 71% 235M/332M [00:05<00:02, 47.3MB/s] 72% 240M/332M [00:06<00:02, 48.0MB/s] 74% 245M/332M [00:06<00:01, 48.4MB/s] 75% 250M/332M [00:06<00:01, 48.8MB/s] 77% 254M/332M [00:06<00:01, 48.8MB/s] 78% 259M/332M [00:06<00:01, 49.4MB/s] 79% 264M/332M [00:06<00:01, 49.2MB/s] 81% 269M/332M [00:06<00:01, 49.7MB/s] 82% 274M/332M [00:06<00:01, 49.5MB/s] 84% 278M/332M [00:06<00:01, 49.8MB/s] 85% 283M/332M [00:06<00:01, 49.6MB/s] 87% 288M/332M [00:07<00:00, 49.6MB/s] 88% 293M/332M [00:07<00:00, 49.3MB/s] 89% 297M/332M [00:07<00:00, 49.6MB/s] 91% 302M/332M [00:07<00:00, 49.5MB/s] 92% 307M/332M [00:07<00:00, 49.7MB/s] 94% 312M/332M [00:07<00:00, 49.7MB/s] 95% 317M/332M [00:07<00:00, 49.9MB/s] 97% 321M/332M [00:07<00:00, 49.7MB/s] 98% 326M/332M [00:07<00:00, 50.0MB/s]100% 331M/332M [00:08<00:00, 49.7MB/s]100% 332M/332M [00:08<00:00, 43.4MB/s]
[2021-12-22 17:45:00,485] {subprocess.py:89} INFO - Testing 0 2263
[2021-12-22 17:45:00,601] {subprocess.py:89} INFO - /opt/conda/lib/python3.8/site-packages/torch/nn/functional.py:3103: UserWarning: The default behavior for interpolate/upsample with float scale_factor changed in 1.6.0 to align with other frameworks/libraries, and now uses scale_factor directly, instead of relying on the computed output size. If you wish to restore the old behavior, please set recompute_scale_factor=True. See the documentation of nn.Upsample for details.
[2021-12-22 17:45:00,602] {subprocess.py:89} INFO -   warnings.warn("The default behavior for interpolate/upsample with float scale_factor changed "
[2021-12-22 17:45:04,691] {subprocess.py:89} INFO - Testing 1 2264
[2021-12-22 17:45:09,390] {subprocess.py:93} INFO - Command exited with return code 0
[2021-12-22 17:45:09,405] {taskinstance.py:1270} INFO - Marking task as SUCCESS. dag_id=data_pipeline1, task_id=super_resolution, execution_date=20211220T003300, start_date=20211222T084446, end_date=20211222T084509
[2021-12-22 17:45:09,440] {local_task_job.py:154} INFO - Task exited with return code 0
[2021-12-22 17:45:09,469] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
